speech_optimization.py

```diff
@@ -105,9 +106,13 @@ def get_acceptable_messages(author, channel):
     
     # Only returns mantra if channels is hexcorp-repetitions; else it returns nothing
     if channel == REPETITIONS:
+
+        print("ADDING ADDITIONAL MANTRA:")
+        print(Mantras.current_mantra)
+
```

Comment:
Beware! `print()` leads to problems when the server is running for some time. Please always use the logger.